### PR TITLE
Pools detection fix for CentOS

### PR DESCRIPTION
**Pull request by LeoShivas**
Replace $(NF-1) by $(NF).
I don't know, but on my CentOs server, the commande "lsof -p 1179 2>/dev/null | grep unix | head -1" gives me :
php-fpm 1179 php-fpm   24u  unix 0xffff8f77b774dc00       0t0 8958617 /local/php/sockets/php7_mediawiki.sock

So, the socket is in the last row, not in the before last.

### CODE REVIEW

**Comment by rvalitov** [comment]
Thank you for reporting this. In Debian I have the following:
```
php-fpm7. 25897              root    9u     unix 0x000000006509e31f        0t0   58381847 /run/php/php7.3-fpm.sock type=STREAM
```

I can't merge your PR now, because it will break this script in Debian. There should be a universal approach that will work for all cases. Do you have an idea and desire to provide such solution?

**Comment by LeoShivas** [comment]
I will look at this.

**Comment by rvalitov** [comment]
Thank you very much. We should also keep in mind about TCP #8

**Comment by LeoShivas** [comment]
Replace $(NF-1) by $(9) ?

**Comment by rvalitov** [comment]
Good idea! I think we don't need the parentheses here, simply `$9`

**Comment by LeoShivas** [comment]
Yes, and for TCP case, you can write :
`lsof -p 1179 2>/dev/null | grep -E "unix|TCP" | head -1 | awk '{ print $9}'`

**Comment by rvalitov** [comment]
Thank you, for TCP I already published a commit with double `-e` expressions. It should be more universal than `-E` which sometimes may require `egrep` instead of `grep`.

File: zabbix/zabbix_php_fpm_discovery.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ while IFS= read -r line
 do
     POOL_PID=`printf '%s\n' "${PS_LIST[@]}" | $S_GREP "php-fpm: pool $line" | $S_HEAD -1 | $S_AWK '{print $1}'`
     if [[ ! -z $POOL_PID ]]; then
-        POOL_SOCKET=`$S_LSOF -p $POOL_PID 2>/dev/null | $S_GREP unix | $S_HEAD -1 | $S_AWK '{ print $(NF-1)}'`
+        POOL_SOCKET=`$S_LSOF -p $POOL_PID 2>/dev/null | $S_GREP unix | $S_HEAD -1 | $S_AWK '{ print $(NF)}'`
```
Comment (rvalitov) [inline zabbix/zabbix_php_fpm_discovery.sh]: I think the following should work:
```
POOL_SOCKET=`$S_LSOF -p $POOL_PID 2>/dev/null | $S_GREP -e unix -e TCP | $S_HEAD -1 | $S_AWK '{ print $9}'`
```

**Pull request merged**